avc/package.xml

```diff
@@ -44,5 +46,10 @@
   <run_depend>image_transport</run_depend>
   <run_depend>message_generation</run_depend>
   <run_depend>message_runtime</run_depend>
+  <run_depend>image_transport</run_depend>
```

Comment:
@barulicm these were the run and build depends that prevented me from being able to launch the nodelets.

Reply:
:+1: No idea why these didn't show up during my last review.